documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+By using NLX certificates for signing and information retrieval, the waardepapieren service will align with REGULATION (EU) No 910/2014 for a large extend, the rest of which is to be deferred to the ongoing development of Discipl like as with ETSI TS 103 171.
+
+The Waardepapieren solution should adhere to EN 301 549 / WCAG 2.0 though this is largely dependent on the integration of the solution in the user interfaces at the side of the issuer and not the solution itself. The project does involve the implementation of a reference validator app  / wallet however which must adhere to EN 301 549 / WCAG 2.0.
+
+As said, the solution also aims at a W3C verifiable credential compliant solution. This compliancy is deferred to the ongoing development of Discipl however. The solution adheres to the main mandatory concepts of this and related standards however.
+
+## Organisational layer
+
+The following stakeholders are identified:
+
+- issuer : automated (abundance) service available at location of issuer and/or through its website
+- clerk : employee that helps a client at location of issuer, services, identifies and authorizes the client manually
+- authorization service : automated service that identifies and authorizes client when client self services him/herself and accesses issuer through website
+- client : person in need of verifiable information required by a validator. It also becomes a holder of the resulting signed information.
+- validator : actor that needs to receive / verify information from client
+- administrator : actor that controls and monitors the automated issuer service
+
+An example of the essential conversation between real people as actors that would take place
+in the use cases the waardepapieren solution facilitates is given below:
+
+![](images/vcseq0.png)
+
+In essence this is the very proces that also takes place when retrieving a proof through the waardepapieren solution as can be seen below in the other verifiable claim sequence diagrams.
+
+![](images/vcseq.png)
+
+## Information layer
+
+All information is temporarily held in verifiable claim channels on the so called discipl ephemeral platform (http://github.com/discipl/discipl-core-ephemeral) in memory. This platform will enforce access rights denoted by the channel owner in specific allow claims within the channel, so by default all channels are only accessible to the channel owner holding a corresponding private key created and temporarily held by the channel owner.Of course the party hosting the ephemeral server and it's administrator needs to be trusted also, but as this is the issuer as source of the information it is by definition a party already trusted with this information by the client.
```

Comment:
owner.Of course the -> owner. Of course, the
needs to -> need to 

"but as this is the issuer as source of the information it is by definition a party already trusted with this information by the client"

This is kind of difficult to parse, maybe replace source with "provider" and/or split up into multiple sentences?

Reply:
changed source into provider